package.json

```diff
@@ -43,7 +43,7 @@
     "@vue/cli-plugin-router": "~4.5.0",
     "@vue/cli-plugin-typescript": "~4.5.0",
     "@vue/cli-plugin-unit-jest": "~4.5.0",
-    "@vue/cli-service": "~4.5.0",
+    "@vue/cli-service": "4.5.17",
```

Comment:
We will not make any changes to package.json file until we add a new package or we update version for some package. Please revert this change as I think this version upgrade is not needed and also revert the change for package-lock.json file

Reply:
Ok Sir I will revert the changes.